demo/full/scripts/components/GenerateLinkURL.tsx

```diff
@@ -9,6 +10,20 @@ function GeneratedLinkURL({
 }: {
   url: string|undefined|null;
 }): JSX.Element {
+
+  const [hasBeenCopied, setHasBeenCopied] = React.useState(false);
+  const onCopyURLtoClipboard = () => {
+    setHasBeenCopied(true);
+    setTimeout(() => setHasBeenCopied(false), 3000);
```

Comment:
If clicking multiple times in a row (i use my computers nervously), it will be strange to have that check switched to false fastly

Reply:
The button being disable should prevent the callback to be run more than once, or is it not the case?